Fireworks/core/message.py

```diff
@@ -349,9 +314,47 @@ def __contains__(self, item):
         else:
             return False
 
+    def __getattr__(self, name):
+
+        if name in self.df.keys():
+            return self.df.__getattribute__(name)
+        elif name in self.tensor_message.keys():
+            return self.tensor_dict.__getattr__(name)
+        else:
+            raise AttributeError("This message has no attribute or column named {0}.".format(name))
+
     def __repr__(self):
         return "Message with \n Tensors: \n {0} \n Metadata: \n {1}".format(self.tensor_message, self.df)
 
+    def to(self, method, *args, **kwargs):
```

Comment:
can you make it more explicit what are args and kwargs? As the saying goes, friends don't let other friends use kwargs :)